tests/core/test_stellargraph.py

```diff
@@ -42,6 +42,25 @@ def create_graph_1(is_directed=False, return_nx=False):
 
 def example_benchmark_graph(
     feature_size=None, n_nodes=100, n_edges=200, n_types=4, features_in_nodes=True
+):
+    node_ids = np.arange(n_nodes)
+    edges = pd.DataFrame(
+        np.random.randint(0, n_nodes, size=(n_edges, 2)), columns=["source", "target"]
+    )
+
+    if feature_size is None:
+        features = []
+    else:
+        features = np.ones((n_nodes, feature_size))
+
+    all_nodes = pd.DataFrame(features, index=node_ids)
+    nodes = {ty: all_nodes[node_ids % n_types == ty] for ty in range(n_types)}
```

Comment:
The 'types' here are ints; would string be more typical? (maybe no real difference due to the remapping)

Reply:
This is matching the existing behaviour of the benchmark. I suspect strings probably are more typical but I don't want to change it here, and I don't think it will make much difference.